op-service/txmgr/txmgr.go

```diff
@@ -111,14 +72,36 @@ type ETHBackend interface {
 // SimpleTxManager is a implementation of TxManager that performs linear fee
 // bumping of a tx until it confirms.
 type SimpleTxManager struct {
-	Config  // embed the config directly
+	cfg     Config // embed the config directly
 	name    string
 	chainID *big.Int
 
 	backend ETHBackend
 	l       log.Logger
 }
 
+// NewSimpleTxManager initializes a new SimpleTxManager with the passed Config.
+func NewSimpleTxManager(name string, l log.Logger, cfg Config) *SimpleTxManager {
+	if cfg.NumConfirmations == 0 {
+		panic("txmgr: NumConfirmations cannot be zero")
+	}
```

Comment:
Maybe also add this check already to `CLIConfig.Check` so it's caught earlier.

Reply:
done